apps/super-app/package.json

```diff
@@ -27,8 +27,8 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "lint": "eslint .",
-    "format": "prettier --write 'src/**/*.{tsx,js,ts}'",
-    "format:check": "prettier --check 'src/**/*.{tsx,js,ts}'"
+    "format": "prettier --write src/**/*.{tsx,js,ts}",
+    "format:check": "prettier --check src/**/*.{tsx,js,ts}"
```

Comment:
@glencbz the command didn't work for me when there's `'`. Were they added for a specific purpose?

Reply:
Adding `'` quotes would prevent the shell from expanding the glob pattern instead of passing the glob directly to `prettier`. I don't recall if this is actually needed or not, but it seemed more portable to me to rely on the tool's globbing instead of the shell's.

What platform are you running this on, and what is the error message you get? Also, did you invoke this via `make format`?

Reply:
I was using `non run format` and it didn't run 😅 I'll remove it first if it's gonna affect CI, and try with `make`!

Reply:
> What platform are you running this on, and what is the error message you get?

What are the answers to these questions? I can help troubleshoot.

>  I'll remove it first if it's gonna affect CI, and try with make

I don't think it'll affect CI, but yes, let's remove it since we don't really understand why we need to make this change right now.

Reply:
> What are the answers to these questions? I can help troubleshoot.

`npm run format` ah. Directly in bash.

Reply:
> What platform are you running this on, and what is the error message you get?

I'd still like to know the answers to these questions. `npm run format ah. Directly in bash.` doesn't give me enough troubleshooting info.

> I don't think it'll affect CI, but yes, let's remove it since we don't really understand why we need to make this change right now.

We wanted to undo this change, right?